structuralcodes/sections/specific.py

```diff
@@ -0,0 +1,446 @@
+""" Specific class section implementation"""
+
+from __future__ import annotations
+
+import typing as t
+import structuralcodes.core._section_results as s_res
+from shapely import Polygon
+from structuralcodes.sections._generic import GenericSection
+from structuralcodes.geometry import SurfaceGeometry
+from structuralcodes.materials.concrete import Concrete
+
+class RectangularSection(GenericSection): 
+    """This is the specific implementation of a rectangular class section.
+
+    The section is a 2D geometry where Y axis is horizontal while Z axis is
+    vertical.
+
+    The moments and curvatures around Y and Z axes are assumed positive
+    according to RHR.
```

Comment:
We should also state that we create the rectangle centered on the origin?

Reply:
Agree and changed